### DISCUSSION TITLE
Move to a better place

### DISCUSSION THREAD

**Discussion by SoheilKhanalipoor** [celzero/rethink-app#289] [General]
Unfortunately, I have to say that you have put this program in a bad place! GitHub!
This app deserves places like: Codeberg, Gitea, GitLab (Instances), and wherever better…
GitHub boycotts users, removes apps, and actually says "Whatever I want is going to be saved and whatever I don't need Is going to be deleted." The app has not paid attention to users' requests for RTL support for years, and says this is not a priority!
Free software should be free somewhere, which GitHub does not have!
Aren't these enough to leave GitHub?

**Comment by ignoramous** [comment]
> Aren't these enough to leave GitHub?

Yes, they are. Unless we start making money, we cannot switch to using our own instances to host software, or afford to pay other hosting services.

You make valid points. Codeberg and Gitea look enticing. Right now, tooling for some of the (`golang` and `JavaScript`) packages _require_ GitHub or _GitLab_, but that can be overhauled. Let's see.

**Reply by ignoramous** [reply]
Read also: https://hugo.barrera.io/journal/2022/04/29/open-source-platforms-for-open-source-software/

**Comment by shuvashish76** [comment]
Related article (for discussion purpose):
- https://sfconservancy.org/blog/2022/jun/30/give-up-github-launch/
- https://giveupgithub.org/
